frontend/fingertips-frontend/components/molecules/Pill/index.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { Paragraph } from 'govuk-react';
+import { typography } from '@govuk-react/lib';
+import React from 'react';
+import styled from 'styled-components';
+
+interface PillProps {
+  selectedFilterName: string;
+}
+
+const StyledDivContainer = styled('div')({
+  backgroundColor: 'white',
+  border: '2px black solid',
+  padding: '10px 20px',
+  maxWidth: '40%',
+  margin: '5px',
+  display: 'flex',
+});
+
+const StyledFilterNameDiv = styled('div')({
+  maxWidth: '80%',
+  wordWrap: 'break-word',
+});
+
+const StyledIconDiv = styled('div')({
+  marginLeft: 'auto',
+  display: 'flex',
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+});
+
+const StyledParagraph = styled(Paragraph)(
+  {
+    marginBottom: '0',
+  },
+  typography.font({ size: 16 })
+);
+
+const StyledIconSvg = styled('svg')({
+  cursor: 'pointer',
+});
+
+export function Pill({ selectedFilterName }: Readonly<PillProps>) {
+  return (
+    <StyledDivContainer data-testid="main-container">
```

Comment:
can we call the data-testid `pill-container` just so it doesn't get confusing the actual main container we have for the page.